### ISSUE TITLE
CI: Arch Linux build is broken

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#160]
e.g. https://github.com/quick-lint/quick-lint-js/runs/1844561387

```shell
$ mkdir build ; cd build ;  cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug .. ; cd "$OLDPWD"
cmake: /usr/lib/libc.so.6: version `GLIBC_2.33' not found (required by /usr/lib/libstdc++.so.6)

$ /usr/bin/docker exec  3faad28777704b17db548301aa0f4a79e253e6d1f7820d542a57fb1fc4e35ff7 sh -c "cat /etc/*release | grep ^ID"
/__e/node12/bin/node: /usr/lib/libc.so.6: version `GLIBC_2.33' not found (required by /usr/lib/libstdc++.so.6)
```

**Comment by strager** [comment]
It seems that others have encountered this issue too:

* https://bbs.archlinux.org/viewtopic.php?id=263379
* https://www.reddit.com/r/archlinux/comments/lqlsd7/archlinux_docker_image_error_failed_to_initialize/

I tried one of the workarounds (using an older Docker image), but it didn't fix the issue.

**Comment by strager** [comment]
`-Syu` seems to fix the issue, at least on my machine.

**Comment by strager** [comment]
Fixed in commit 52b0d0c.
